modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/IgniteCustomAssigmentsRules.java

```diff
@@ -178,16 +173,20 @@ private IgniteCustomAssigmentsRules(
 
```

Comment:
why do you have special treatment of TIME WITH LOCAL TIME ZONE?

Reply:
WITH LOCAL is not supported for now 
https://issues.apache.org/jira/browse/IGNITE-19274, (TIME|TIMESTAMP)_WITH_LOCAL are direct copy from appropriate SqlTypeAssignmentRule

Reply:
how mentioned ticket relates to the given patch?